config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
why would we need to copy it around at all ? here is what i have in mind:

- a core or kernel upgrade comes in ... snapd uses abootimg to modify the running cmdline and adds snap_mode=try and the respective snap_try_* variable to it, then reboots into recovery.
- an initrd script inside recovery parses the cmdline, for core updates it moves the snap_try_core content to snap_core and sets snap_mode=trying, for kernel it additionally flashes the boot.img found inside the mentioned core snap, then reboots.
- if the boot succeeds, snapd cleans up the cmdline
- if the boot fails we reboot into recovery, the script finds snap_mode=trying and reverts everything.
the common place to exchange all information between recovery and normal boot is the cmdline of the actual boot partition, there is no need for any additional place. any flashing or extracvting of a boot.img file from a kernel snap can happen inside the recovery.